File: src/main/core/io/github/kloping/Mcore/java/Plugins/PluginLoader.java
package io.github.kloping.Mcore.java.Plugins;

import net.mamoe.mirai.console.terminal.MiraiConsoleTerminalLoader;

/**
 * @author github-kloping
 */
public class PluginLoader {
    public static void load(String[] args) {
        MiraiConsoleTerminalLoader.main(args);
//        try {
//            PrintStream oldPs = System.out;
//            PrintStream oldErr = System.err;
//            System.setErr(oldErr);
//            System.setOut(oldPs);
//        } catch (Exception e) {
//            e.printStackTrace();
//        }
    }
}

File: src/main/core/io/github/kloping/Mcore/kotlin/Plugins/PluginLoader.kt
package io.github.kloping.Mcore.kotlin.Plugins

import net.mamoe.mirai.console.terminal.MiraiConsoleTerminalLoader.main

/**
 * @author github-kloping
 */
object PluginLoader {
    fun load(args: Array<String>?) {
        try {
            val oldPs = System.out
            val oldErr = System.err
            main(args!!)
            System.setErr(oldErr)
            System.setOut(oldPs)
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }
}